pennylane_lightning/core/src/simulators/lightning_qubit/measurements/MeasurementKernels.hpp

```diff
@@ -296,6 +297,86 @@ namespace PUtil = Pennylane::Util;
 
 namespace Pennylane::LightningQubit::Measures {
 
+/**
+ * @brief Generate samples using the alias method.
+ *
+ * @tparam PrecisionT Precision data type
+ */
+template <typename PrecisionT> class discrete_random_variable {
+  private:
+    static constexpr std::size_t default_index =
+        std::numeric_limits<std::size_t>::max();
+    const std::vector<std::pair<double, std::size_t>> bucket_partners_;
+    std::mt19937 &gen_;
+    const std::size_t n_probs;
+    mutable std::uniform_real_distribution<PrecisionT> distribution{0.0, 1.0};
```

Comment:
You may want to extend the naming convention for private class variables here too,
```suggestion
    const std::size_t _n_probs;
    mutable std::uniform_real_distribution<PrecisionT> _distribution{0.0, 1.0};
```